### PR TITLE
update/valheim-vanilla

### PR DESCRIPTION
**Pull request by Attackwave**
# Description

Added new command line arguments:
- public server y/n (public)
- backup settings (saveinterval, backups, backupshort, backuplong)

## Checklist for all submissions

* [X] Have you followed the guidelines in our [Contributing document](https://github.com/parkervcp/eggs/blob/master/CONTRIBUTING.md)?
* [X] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [X] Have you tested and reviewed your changes with confidence that everything works?
* [X] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### CODE REVIEW

**Comment by Attackwave** [comment]
image
image

File: game_eggs/steamcmd_servers/valheim/valheim_vanilla/egg-valheim.json [hunk 1]
```diff
@@ -112,14 +122,54 @@
...
             "env_variable": "ENABLE_CROSSPLAY",
             "default_value": "1",
             "user_viewable": true,
             "user_editable": true,
             "rules": "boolean",
             "field_type": "text"
         },
+        {
+            "name": "Backup Interval",
+            "description": "Change how often the world will save in seconds.",
```
Comment (parkervcp) [inline game_eggs/steamcmd_servers/valheim/valheim_vanilla/egg-valheim.json:135]: You want to make a note that this is 30 minutes by default

**Review by parkervcp — CHANGES_REQUESTED** [review]

**Comment by Attackwave** [comment]
Variables/Descriptions synced and updated.

**Review by gOOvER — APPROVED** [review]
works without problems, as already mentioned in Discord

**Review by QuintenQVD0 — APPROVED** [review]

**Pull request merged**